.github/workflows/clair-in-ci.db.yaml

```
@@ -70,5 +61,4 @@ jobs:
         if: ${{ github.event.client_payload.pr_event != 'pull_request' }}  # don't push image from PR
         id: push-image
         run: |
-          docker tag ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.TO_TEST }} ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.LATEST_TAG }}
-          docker push ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.LATEST_TAG }}
+          skopeo copy ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.TO_TEST }} ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.LATEST_TAG }}
```

is skopeo installed there?

Add --all param to skopeo, so it copies everything including manifests lists (if exists, but eventually they will exist)

Also we should push to `v1` tag as with clamav and slowly deprecate usage of latest
```suggestion
          skopeo copy --all ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.TO_TEST }} ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.LATEST_TAG }}
                    skopeo copy --all ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.TO_TEST }} ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.MAJOR_VERSION_TAG }}
```